tests/integration/test_contains.py

```diff
@@ -12,19 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import pytest
-
-import cunumeric as num
 
+from functools import reduce
 
-def test_True():
-    x = num.array([1, 2, 3, 4, 5])
-    assert 4 in x
+import pytest
+from utils.generators import mk_seq_array
 
+import cunumeric as num
 
-def test_False():
-    x = num.array([1, 2, 3, 4, 5])
-    assert 6 not in x
+DIM = 128
+NO_EMPTY_SIZES = [
+    (DIM,),
+    (1, DIM),
+    (DIM, 1),
+    (DIM, DIM),
+    (DIM, 1, 1),
+    (1, DIM, 1),
+    (1, 1, DIM),
+    (DIM, DIM, DIM),
+]
+
+
+@pytest.mark.parametrize("size", NO_EMPTY_SIZES)
+def test_int(size):
+    arr = mk_seq_array(num, shape=size)
+    max_data = reduce(lambda x, y: x * y, size)
+    assert max_data - 2 in arr
+    assert max_data + 2 not in arr
+
+
+@pytest.mark.parametrize("size", NO_EMPTY_SIZES)
+def test_complex(size):
+    arr = mk_seq_array(num, shape=size) + mk_seq_array(num, shape=size) * 1.0j
+    max_data = reduce(lambda x, y: x * y, size)
+    assert (max_data - 2) + (max_data - 2) * 1.0j in arr
+    assert (max_data + 2) + (max_data + 2) * 1.0j not in arr
+    assert (max_data - 2) + max_data * 1.0j not in arr
+    assert max_data - 2 not in arr
```

Comment:
These tests are good but a little opaque. It might be good to retain a few basic/simple tests that just test `in` and `not in` with some small concrete arrays. 

Additionally for these tests with `mk_seq_array`, I think we could be a little more comprehensive based on things we can assume about `mk_seq_array`:


* check -1 is not in arr
* check 0 is not in arr
* check 1 is in arr
* check a value(s) in the middle are in arr
* check that `max_data` is in arr
* check that `max_data` +1 is in arr

Reply:
fixed